packages/beacon-node/src/network/reqresp/types.ts

```diff
@@ -201,6 +250,25 @@ export const reqRespBlockResponseSerializer = {
   },
 };
 
+export const reqRespLightClientUpdateSerializer = {
+  serialize: (chunks: LightClientUpdate[]): Uint8Array => {
```

Comment:
Is there a more idiomatic way to serialise this with ssz?

Reply:
I think we only want to serialize a single chunk at a time here
From the spec:
> The response MUST consist of zero or more response_chunk. Each successful response_chunk MUST contain a single LightClientUpdate payload.